git_remote_s3/remote.py

```diff
@@ -159,6 +160,18 @@ def cmd_push(self, args: str) -> str:
             if remote_to_remove:
                 self.s3.delete_object(Bucket=self.bucket, Key=remote_to_remove)
 
+            if self.uri_scheme == "s3+zip":
```

Comment:
Use an enum or a constant instead of a string, ie `if self.uri_scheme == URI_SCHEME_S3_ZIP:` or `if self.uri_scheme == UriScheme.S3_ZIP:`. 
In you opt for the letter ensure that `self.uri_scheme` is also an Enum. 
You might need to update the tests too.

Reply:
I moved from my famous magic strings to Enums.